package.json

```
@@ -64,8 +64,8 @@
     "mocha": "5.2.0",
     "mocha-lcov-reporter": "1.3.0",
     "nyc": "13.1.0",
-    "sinon": "7.2.0",
     "sequelize-cli": "5.3.0",
-    "sinon-chai": "3.3.0"
+    "sinon": "7.2.0",
+    "sinon-chai": "^3.3.0"
```

Good Job @andrewinsoul.
Please do always ensure you reference an `exact` version in your dependencies. or install your modules with `npm i packagename -E`  so as to remove the caret symbol( ^ ) automatically

```suggestion
    "sinon-chai": "3.3.0"
```

---

Thanks @shegsteham let me fix that